pkg/issuer/acme/http/service.go

```diff
@@ -113,6 +112,15 @@ func buildService(crt *v1alpha1.Certificate, ch v1alpha1.ACMEOrderChallenge) *co
 			Selector: podLabels,
 		},
 	}
+
+	// http01 config is still optional, thus checking for presence and setting previous default
+	if issuer.GetSpec().ACME.HTTP01 == nil {
+		service.Spec.Type = corev1.ServiceTypeNodePort //TODO drop this line in future to use Kubernetes' default
```

Comment:
We are unlikely to drop this in future - NodePort makes the most sense for us to cover the real wide variety of ingress controller implementations out there. I think we should remove the TODO comment 😄